packages/react-magma-dom/src/components/Tooltip/index.tsx

```diff
@@ -239,26 +256,31 @@ export const Tooltip = React.forwardRef<any, TooltipProps>((props, ref) => {
     >
       {tooltipTrigger}
       {isVisible && (
-        <div
-          ref={setPopperElement}
-          style={combinedTooltipStyles}
-          {...attributes.popper}
-        >
+        <div ref={refs.setFloating} style={combinedTooltipStyles}>
+          <FloatingArrow
+            ref={arrowElement}
+            data-testid="tooltipArrow"
```

Comment:
I prefer testIds to be unique. Can we use `${testId}-arrow` or something like that?